packages/mira/tasks/latex_to_amr.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import json
+import traceback
+
+from taskrunner import TaskRunnerInterface
+from sympy.parsing.latex import parse_latex
+from mira.sources.sympy_ode import template_model_from_sympy_odes
+from mira.modeling.amr.petrinet import template_model_to_petrinet_json
+
+def cleanup():
+    pass
+
+def main():
+    exitCode = 0
+
+    try:
+        taskrunner = TaskRunnerInterface(description="Latex to SymPy")
```

Comment:
If you have to do case right for SymPy, LaTeX should also be.

```suggestion
        taskrunner = TaskRunnerInterface(description="LaTeX to SymPy")
```